src/libraries/shims/ApiCompat.proj

```diff
@@ -111,7 +111,7 @@
       <_previousNetCoreAppBaselineParam Condition="'$(UpdatePreviousNetCoreAppBaseline)' == 'true'">&gt; &quot;$(_previousNetCoreAppBaselineFile)&quot;</_previousNetCoreAppBaselineParam>
     </PropertyGroup>
  
-    <Error Condition="'$(NetCoreAppCurrent)' != 'net5.0'" Text="Update value of PreviousNetCoreApp" />
+    <Error Condition="'$(NetCoreAppCurrent)' != 'net6.0'" Text="Update value of PreviousNetCoreApp" />
```

Comment:
You need to update the values
https://github.com/dotnet/runtime/blob/7cbc26c13fd41d7fa879bdcbe44f0a4513f24be0/src/libraries/shims/ApiCompat.proj#L8-L10

To test against netcoreapp5.0 API.

Reply:
I was waiting to until we release the stable 5.0 stable bits.

Reply:
We don't need stable bits.  We can switch to RC and file an issue to update it.  Technically we could change this is soon as be branch for release and move to v.Next.  It protects us from breaking API that was added in the last release.